weather_app/models.py

```diff
@@ -0,0 +1,41 @@
+from __future__ import unicode_literals
+from django.db import models
+import uuid
+
+""" Custiom time stamp model """
+class CustomTimeStampModel(models.Model):
+    created_at = models.DateTimeField(auto_now_add=True)
+    modified_at = models.DateTimeField(auto_now=True)
+    is_active = models.BooleanField(default=True)
+
+    """ using to set class abstract for to create a model """
```

Comment:
wrong meaning of meaning of model in docstring its leading to misconception of abtract models